### DISCUSSION TITLE
Table Widget Full Span

### DISCUSSION THREAD

**Discussion by juangdnar** [filamentphp/filament#5192] [Help] [answered]
In a table widget, how can I make the width of the table cover the entire section? sample image attached:

![Sin título](https://user-images.githubusercontent.com/115037170/205940424-668fa7cc-5539-421c-b211-5b772778f291.png)

**Comment by danharrin** [comment, marked as answer]
`protected int | string | array $columnSpan = 'full';`

**Reply by danharrin** [reply]
https://filamentphp.com/docs/2.x/admin/dashboard/getting-started#customizing-widget-width

**Reply by juangdnar** [reply]
Thanks a lot!!! :)
